README.md

```diff
@@ -241,12 +241,17 @@ export const getServerSideProps: GetServerSideProps<Data> = async (ctx) => {
     // userId: "123" // etc
   };
 
-  const { toggles } = await getFrontendFlags({ context }); // Use Proxy/Frontend API
-  const flags = flagsClient(toggles);
+  const definitions = await getDefinitions(); // Uses UNLEASH_SERVER_API_URL
+  const { toggles } = evaluateFlags(definitions, context);
+
+  const flags = flagsClient(toggles); // instantiates a static (non-syncing) unleash-proxy-client
 
   return {
     props: {
       isEnabled: flags.isEnabled("nextjs-example"),
+      // Or you can skip the flagsClient and access the toggles directly, but this will skip impression logging
+      // and other client logic from unleash-proxy-client
+      isEnabled: toggles.find((t) => t.name === 'FEATURE_NAME').enabled
```

Comment:
This can be misleading. It's not a result of evaluation. This is from definitions. Feature can be enabled in definitions and still evaluate strategies to `false`.

Reply:
Ooooh! Okay, I was confused about that - I'll just remove this...